File-Formats/PDF/Working-with-Tagged-PDF.md

```diff
@@ -2906,3 +2906,184 @@ Follow the below steps to validate the tagged PDF information in Adobe Acrobat:
 The following screenshot shows the Accessibility checker dialog box.
 
 ![TaggedPDF](TaggedPDF_images/TaggedPDF_img0.png) 
+
+## Get the accessibility tags
+
+You can extract the existing tag details by using the [StructureElement](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedDocument.html#Syncfusion_Pdf_Parsing_PdfLoadedDocument_StructureElement) property available on the [PdfLoadedDocument](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedDocument.html) class. The following code explains this.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+//Load the existing PDF document.
+PdfLoadedDocument document = new PdfLoadedDocument("TaggedPDF.pdf");
+//Get the structure element root from the document.
+PdfStructureElement rootElement = document.StructureElement;
+//Get the first page from the document.
+PdfLoadedPage loadedPage = document.Pages[0] as PdfLoadedPage;
+//Get the structure elements associated with the page.
+PdfStructureElement[] pageElements = loadedPage.StructureElements;
```

Comment:
This should be split as a separate topic.

You can also extract the accessibility tags page-wise with the help of the StructureElement property available on the PdfLoadedPage class. The following code illustrates this.

Reply:
Hi @chinnumuniyappan,

The suggested changes has been updated.

Regards,
Anantha Gokula Raman J